chaoscenter/web/src/views/CreateNewUser/CreateNewUser.tsx

```diff
@@ -41,6 +42,9 @@ export default function CreateNewUserView(props: CreateNewUserViewProps): React.
       {
         onSuccess: () => {
           handleClose();
+        },
+        onError: (err: any) => {
+          showError(err.errorDescription);
```

Comment:
Is this tested? Can you check if it is `err.message` or `err.description?`

Reply:
Yes, it's already tested. The err object does not have a field err.message; it is undefined. Instead, it has a field errDescription, which I used. 
Here is the err object as shown in the console:
error: "user_exists"
errorDescription:"This username is already assigned to another user"